### PR TITLE
Added postcode validation for Sweden and Turkey.

### PR DESCRIPTION
**Pull request by onurersel**
### Summary
Added postcode validation for Sweden and Turkey.
Added localization for Turkish.

### CODE REVIEW

**Comment by madhikarma** [comment]
Hello @onurersel thank you for this contribution! We'll try to give a code review soon but looking good so far. cc @axellundback

**Comment by onurersel** [comment]
Cheers @madhikarma! 🎉

File: Sources/Conditions/PostcodeCondition.swift [hunk 1]
```diff
@@ -9,10 +9,12 @@
 import Foundation
 
 
-/// Countries that are supported by `PostcodeCondtion`. Each postcode is an ISO 3166-1 alpha-3 country code. There is a `regex` property that returns the regex for validating that country's postcode.
+/// Countries that are supported by `PostcodeCondition`. Each postcode is an ISO 3166-1 alpha-3 country code. There is a `regex` property that returns the regex for validating that country's postcode.
```
Comment (ghost) [inline Sources/Conditions/PostcodeCondition.swift:12]: Good catch!

File: Tests/Unit Tests/Conditions/PostcodeConditionTests.swift [hunk 2]
```diff
@@ -16,48 +16,187 @@ final class PostcodeConditionTests: XCTestCase {
...
     // MARK: - Properties
     
-    let condition       = PostcodeCondition(country: .UnitedKingdom)
+    let conditionGBR        = PostcodeCondition(country: .UnitedKingdom)
+    let conditionSWE        = PostcodeCondition(country: .Sweden)
+    let conditionTUR        = PostcodeCondition(country: .Turkey)
     
     
-    // MARK: - Test Success
+    // MARK: - GBR - Test Success
```
Comment (ghost) [inline Tests/Unit Tests/Conditions/PostcodeConditionTests.swift:24]: This is a good start. I think I'd like to refactor these tests in a future PR to help remove the redundancy. 🤔

**Review by ghost — APPROVED** [review]
Nice work!

File: Tests/Unit Tests/Conditions/PostcodeConditionTests.swift (hunk 2 above)
Comment (onurersel) [reply Tests/Unit Tests/Conditions/PostcodeConditionTests.swift:24]: Good thinking. I applied the same set of tests to just different conditions there, so there's plenty of space to for refactoring. 👍

**Comment by onurersel** [comment]
Thanks @aamctustwo !

**Pull request merged**